_locales/en/messages.json

```diff
@@ -2300,9 +2294,6 @@
 	"choose_your_ghostery_privacy_plan": {
 		"message": "Choose your Ghostery privacy plan"
 	},
-	"ghostery_basic": {
```

Comment:
We still need this one unless the translation policy on "Basic" has changed from what is currently in the translations ticket.

Reply:
Re-added!